src/components/PlanDeEstudio/Semestre/Materia/Materia.js

```diff
@@ -11,6 +11,8 @@ export default function Materia ({ nums, materia, tec21, clickMateria }) {
         <Row className="nombre-materia p-0 m-0">
           <Col className="p-0 m-0">
             <label>{materia.nombre}</label>
+            <br></br>
+            <label className="unidades">Unidades: {materia.unidades}</label>
```

Comment:
Por si acaso haz la revisión de si la materia contiene unidades.
```suggestion
{materia.unidades && (
  <label className="unidades">Unidades: {materia.unidades}</label>
)}
```

Reply:
Pues en teoría las materias siempre van a traer de cuantas unidades es, entonces esto no sería necesario.